update_version.sh

```diff
@@ -62,25 +62,32 @@ if [[ -z "$patch" ]]; then
 fi
 
 # Build version
-sed_wrapper -i "s/FLB_VERSION_MAJOR  [0-9]/FLB_VERSION_MAJOR  $major/g" "$SCRIPT_DIR"/CMakeLists.txt
-sed_wrapper -i "s/FLB_VERSION_MINOR  [0-9]/FLB_VERSION_MINOR  $minor/g" "$SCRIPT_DIR"/CMakeLists.txt
-sed_wrapper -i "s/FLB_VERSION_PATCH  [0-9]/FLB_VERSION_PATCH  $patch/g" "$SCRIPT_DIR"/CMakeLists.txt
+sed_wrapper -i -E "s/(FLB_VERSION_MAJOR[[:space:]]+)[0-9]+/\1$major/g" "$SCRIPT_DIR"/CMakeLists.txt
```

Comment:
@cosmo0920 did you run this script to generate the updates right?

Reply:
Yes, I used this script to update version. This updated version works perfectly.